models/t_parse.go

```diff
@@ -14,24 +14,6 @@ import (
 //
 // Recommended calling convention:  Process the exceptions first, then use the
 // function for everything else.
-//
-//		  var err error
-//		  switch rType {
-//	   case "MX":
-//	       // MX priority in a separate field.
-//	       if err := rc.SetTargetMX(cr.Priority, target); err != nil {
-//	         return nil, fmt.Errorf("unparsable MX record received from cloudflare: %w", err)
-//	       }
-//		  case "TXT":
-//	       // TXT records are stored verbatim; no quoting/escaping to parse.
```

Comment:
This example disappeared in the new revision.  Please bring it back.

Reply:
If there should be any text which explains its presence, that would be great. Like "For example", or "note to self". But I read the code segment and I am none the wiser.